src/components/main/friends/styles.scss

```diff
@@ -7,13 +7,40 @@
   flex-direction: row;
 
   #content {
+    display: flex;
     flex: 1;
     min-width: 0;
     height: 100%;
     display: inline-flex;
     overflow-x: hidden;
     overflow-y: scroll;
     flex-direction: column;
+
+    .main {
+      display: flex;
+
+      .friends-list {
+        padding: 1rem;
+        flex-grow: 1;
+      }
+
+      .friends-separator {
+        align-items: center;
+        height: 12px;
```

Comment:
I know this was already here but we should peek at seeing if we can remove the calcs on the width somehow and maybe change the margin around to something not pixel defined.

Reply:
I'll create an issue for this now